tests/MyGame/Example/Vec3.php

```diff
@@ -25,49 +25,49 @@ public function init($_i, ByteBuffer $_bb)
     /**
      * @return float
      */
-    public function GetX()
+    public function getX()
```

Comment:
This would break the API. Is this better inline with PHP standards? Are we saying it is ok to break the API now since it requires a new flag to be specified anyway? Maybe document that if so?

Reply:
It is ok, because:
> Note: Function names are case-insensitive, though it is usually good form to call functions as they appear in their declaration.
https://www.php.net/manual/en/functions.user-defined.php

Reply:
@aardappel PSR (PHP Standards Recommendations) enforces camel case. As @LuckyRu said, PHP is case-insensitive so it won't break anything, for now.